includes/scroll-button.html

```diff
@@ -0,0 +1,4 @@
+<div href="" class="arrow-for-scrolling-top" id="js-scroll-top">
+    <span class="arrow"></span>
+</div>
+{% include scripts %}
```

Comment:
Including `scripts` here will duplicate them on some pages - do you mind removing this line? For example, in the `default` layout its already included via `includes/small-footer.html`!

```suggestion
```

Reply:
@JasonEtco 
Thank you for your advice！
I remove the including scripts `{% include scripts %}`